otherlibs/threads/pervasives.ml

```diff
@@ -479,7 +479,7 @@ let really_input_string ic len =
   really_input ic s 0 len;
   bytes_unsafe_to_string s
 
-external bytes_set : bytes -> int -> char -> unit = "%bytes_safe_set"
+external bytes_set : bytes -> int -> char -> unit = "%string_safe_set"
```

Comment:
I don't understand this change.  Shouldn't we use the recommend `%bytes_safe_set` in the stdlib?